docs/quickstart.rst

```diff
@@ -493,23 +497,23 @@ First, let's start up a container with Kafka Connect.  Connect stores all its st
 
```

Comment:
We no longer have to create these topics by hand, so should we remove these steps to keep things as simple as possible?